en-US/mozorg/500.lang

```diff
@@ -0,0 +1,17 @@
+## NOTE: page is at https://www-dev.allizom.org/500/
```

Comment:
Have you tried the URL? This is not a page that can be replicated by URL. You should remove the following line, and explain it in the notes